frontend/src/views/Quiz.vue

```diff
@@ -36,7 +33,7 @@ onBeforeMount(() => {
                 );
 
                 for (const question of response.data.value?.questions || []) {
-                    useFetch<QuizQuestion>(
+                    useFetch<Quizoot.Question>(
```

Comment:
this is why I have `ApiResponse = any` right now. I'll think about a fix

Reply:
I understand